documentation/source/BenchmarkingYoloNAS.md

```diff
@@ -0,0 +1,45 @@
+# Benchmarking YoloNAS
+## Introduction:
+
+YoloNAS is a leading object detection architecture that combines accuracy and efficiency. Using quantization-aware training (QAT), YoloNAS models can be optimized for resource-constrained devices.
+However, to fully tap into its potential, it is crucial to know how to export the quantized model to the INT8 TensorRT (TRT) engine.
+In this blog, we emphasize the significance of this step and provide a concise guide to efficiently exporting a quantized YoloNAS model to the INT8 TRT engine.
+Doing so teaches us how to properly benchmark YoloNAS and understand its full potential.
+
+## Step 1: Export YoloNAS to ONNX
+
+The first step is to export our YoloNAS model to ONNX correctly. Two actions must be taken before we export our model to onnx:
+1. We need to replace our layers with "fake quantized" ones - this happens when we perform post-training quantization or quantization-aware training with SG.
+So nothing to worry about if you performed PTQ/QAT with SG and hold your newly exported ONNX checkpoint.
+2. We must call model.prep_model_for_conversion - this is essential as YoloNAS incorporates QARepVGG blocks. Without this call, the RepVGG branches will not be fused, and our model's speed will decrease significantly!
+Again, nothing to worry about if you have quantized your model with PTQ/QAT with SG, as this is done under the hood before exporting the ONNX checkpoints.
+   
+There are plenty of guides on how to perform PTQ/QAT with SG:
+- [Quantization-aware fine-tuning YoloNAS on custom dataset notebook](https://colab.research.google.com/drive/1yHrHkUR1X2u2FjjvNMfUbSXTkUul6o1P?usp=sharing)
+- [QA/PTQ YoloNAS with configuration files](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/qat_ptq_yolo_nas.md)
+- [QA/PTQ](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/ptq_qat.md)
+
+Suppose we ran PTQ/QAT, then our PTQ/QAT checkpoints have been exported to our checkpoints directory.
+If we plug them into [netron.app](https://netron.app), we can see that new blocks that were not a part of the original network were introduced: the **Quantize/Dequantize** layers - 
+
+<div>
+<img src="images/qdq_yolonas_netron.png" width="750">
+</div>
+This is expected and an excellent way to verify that our model is ready to be converted to Int8 using Nvidia's TesnorRT.
+
+## Step 2: Create TRT Engine
+First, please make sure to [install Nvidia's TensorRT](https://developer.nvidia.com/tensorrt-getting-started).
+TensorRT version >= 8.4 is required.
+We can now use these ONNX files to deploy our newly trained YoloNAS models to production. When building the TRT engine, it is essential to specify that we convert to Int8 (the fake quantized layers in our models will be adapted accordingly); this can be done by running: `trtexec --fp16 --int8 --onnx=your_yolonas_qat_model. onnx.
```

Comment:
use ```
instead of `

Reply:
here, explain that if you ran PTQ or QAT you need the --int8 flag. but if you did not, than the --int8 flag will dramatically degrade the accuracy of your compiled model.
also, explain that by passing --fp16 and --int together, we allow the hybrid quantization (that is, some layers are quantized while others are not)

Reply:
@natan should we add --best as well?

Reply:
You did not tag the correct person :)
AFAIK, `best` is alias for `--int8 --fp16`

Reply:
@ofrimasad fixed in a28a16fead70961f3f7b3f43be4000c9c64ff81d